payments/utils.py

```diff
@@ -0,0 +1,47 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.utils import timezone
+
+from fiobank import FioBank
+
+from konfera.models import Order
+from konfera.models.order import AWAITING, PAID
+
+
+DATE_FORMAT = '%Y-%m-%d'
+
+
+def _is_order_paid(order, payments):
```

Comment:
Payments are important and in this case i think we shall log everything. Please consider adding log:

* payment was found but it is outside of error rate (WARNING)
* payment status has changed (INFO)

There might be monitoring in place which will alerts warnings.